### PR TITLE
Simplified Grid

### PR DESCRIPTION
**Pull request by saicharan-gouru**
Please review my simplified grid component.
Screenshot (195)

### CODE REVIEW

File: Components/SimplifiedGrid/simplified-grid.html [hunk 1]
```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Simplified Grid</title>
+    <link href="/CSS/simplified-grid.css" rel="stylesheet" type="text/css" />
```
Comment (ranitasahaa1230) [inline Components/SimplifiedGrid/simplified-grid.html:9]: You can add a description in your meta tag as well like author details, page description

File: CSS/simplified-grid.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
+@import url('https://fonts.googleapis.com/css2?family=Source+Sans+Pro:ital@1&family=Ubuntu:wght@300&display=swap');
+body {
+    font-family: 'Source Sans Pro', sans-serif;
+    font-family: 'Ubuntu', sans-serif;
```
Comment (ranitasahaa1230) [inline CSS/simplified-grid.css:4]: If this font family is the same for all the features, then you can add this in the main CSS file

**Review by ranitasahaa1230 — APPROVED** [review]
Code can be understood easily
Well documented

File: CSS/simplified-grid.css [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+}
+
+.grids-container {
+    max-width: 500px;
```
Comment (anupkgurung) [inline CSS/simplified-grid.css:8]: rem can be used here instead of px

**Review by anupkgurung — APPROVED** [review]

**Review by Nikhil-Nirmal — APPROVED** [review]

**Review by Veekshith-Thoram — APPROVED** [review]

**Pull request merged**